types/swarm/container.go

```diff
@@ -0,0 +1,74 @@
+package swarm
+
+import "time"
+
+// ContainerSpec represents the spec of a container.
+type ContainerSpec struct {
+	Image           string            `json:",omitempty"`
+	Labels          map[string]string `json:",omitempty"`
+	Command         []string          `json:",omitempty"`
+	Args            []string          `json:",omitempty"`
+	Env             []string          `json:",omitempty"`
+	Dir             string            `json:",omitempty"`
+	User            string            `json:",omitempty"`
+	Mounts          []Mount           `json:",omitempty"`
+	StopGracePeriod *time.Duration    `json:",omitempty"`
+}
+
+// Mount represents a mount (volume).
+type Mount struct {
```

Comment:
There is a little bit of redundancy with `types.Volume`, `types.MountPoint` in current master and what about #258 ? /cc @cpuguy83